lectures/01.intro.md

```diff
@@ -128,14 +129,19 @@ We are recording, aren't we?
 
 --- VERTICAL SLIDE ---
 
-<!-- .slide: data-background="url(resources/01.intro/cryowar_cover.png)" data-background-repeat="no-repeat" data-background-size="cover" data-background-position="center center" class="high-contrast-text" -->
+### Kipi Interactive
+![Kipi Cryowar Heavy](resources/01.intro/CryowarHeavy_Idle.png) 
+
+--- VERTICAL SLIDE ---
 
+<!-- .slide: data-background="url(resources/01.intro/cryowar_cover.png)" data-background-repeat="no-repeat" data-background-size="cover" data-background-position="center center" class="high-contrast-text" -->
 ### Kipi Interactive
 
-* Start-up working on a mobile top-down shooter
-* Founded by Viktor
-* Team of 10
-* UE4-based
+* Independent Game Development Studio
+* Founded by Viktor in 2014
+* Working on Realtime Multiplayer Arena Brawler <a target="_blank" href="https://cryowar.com">Cryowar</a> 

```

Comment:
1. on **a** Realtime...
2. use the markdown syntax for links: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links